src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx

```diff
@@ -321,12 +329,22 @@ const KlimaStakingTransactionBlock = ({
         <Text size={12} marginBottom={2} medium block>
           {option.title}
         </Text>
-        {!!receiveAmount && (
-          <Text size={16} medium>
-            {receiveAmount} {klimaAsset.symbol}
-            {targetAssetPriceUsd && ` · ${formatAmountDisplay(+receiveAmount * targetAssetPriceUsd, '$', 2)}`}
-          </Text>
-        )}
+        <OfferGasPriceContainer>
+          <div>
+            <Text size={12} marginRight={4} color={theme.color?.text?.innerLabel} medium>
+              Gas price:&nbsp;
+            </Text>
+            <Text size={14} marginRight={22} medium inline>
+              {option.gasCostUSD && `$ ${option.gasCostUSD}`}
```

Comment:
```suggestion
              {option.gasCostUSD && `$${option.gasCostUSD}`}
```

Reply:
or maybe consider using `formatAmountDisplay` with `minimumFractionDigits: 2` as per latest product requirements we were asked to show 2 decimals on USD price